contracts/sdk/gummyroll/solita.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const {
+    rustbinMatch,
+    confirmAutoMessageConsole,
+} = require('@metaplex-foundation/rustbin')
+const { spawn } = require('child_process');
+const { Solita } = require('@metaplex-foundation/solita');
+const { writeFile } = require('fs/promises');
+const { fstat, existsSync, realpath, realpathSync } = require('fs');
+
+const PROGRAM_NAME = 'gummyroll';
+const PROGRAM_ID = 'GRoLLzvxpxxu2PGNJMMeZPyMxjAUH9pKqxGXV9DGiceU';
+
+const programDir = path.join(__dirname, '..', '..', 'programs', 'gummyroll');
+const generatedIdlDir = path.join(__dirname, 'idl');
+const generatedSDKDir = path.join(__dirname, 'src', 'generated');
+
+async function main() {
+    const anchorExecutable = realpathSync("../../../deps/anchor/target/debug/anchor");
```

Comment:
Add to list of todos: switch to normal anchor since it builds to solana 1.10 now